Tinder/Models/Instagram.cs

```diff
@@ -1,6 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
 namespace Tinder.Models
 {
     public class Instagram
```

Comment:
This class is used in Profile. Could you confirm that the Instagram object in the recommendations is the same as the one in profile?

Reply:
> This class is used in Profile. Could you confirm that the Instagram object in the recommendations is the same as the one in profile?